src/Microsoft.Health.Dicom.Functions/BlobMigration/CleanupDeletedDurableFunction.cs

```diff
@@ -0,0 +1,32 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Features.BlobMigration;
+using Microsoft.Health.Dicom.Core.Features.ChangeFeed;
+
+namespace Microsoft.Health.Dicom.Functions.BlobMigration;
+
+/// <summary>
+/// Represents the Azure Durable Functions that copy existing DICOM instances to new format.
+/// </summary>
+public partial class CleanupDeletedDurableFunction
+{
+    private readonly IChangeFeedStore _changeFeedStore;
+    private readonly BlobMigrationOptions _options;
+    private readonly BlobMigrationService _blobMigrationService;
+
+    public CleanupDeletedDurableFunction(
+        IChangeFeedStore changeFeedStore,
+       BlobMigrationService blobMigrationService,
+        IOptions<BlobMigrationOptions> configOptions)
+    {
+        _changeFeedStore = EnsureArg.IsNotNull(changeFeedStore, nameof(changeFeedStore));
+        _blobMigrationService = EnsureArg.IsNotNull(blobMigrationService, nameof(blobMigrationService));
+        _options = EnsureArg.IsNotNull(configOptions?.Value, nameof(configOptions));
+        EnsureArg.IsNotNull(_options.RetryOptions, nameof(_options.RetryOptions));
```

Comment:
You can avoid this if you use the data annotation for this member `Required` and read the options `ValidateDataAnnotations` like here: https://github.com/microsoft/dicom-server/blob/e442a6d25e4b9de28c12f7c3abe484d053c5547c/src/Microsoft.Health.Dicom.Functions/Registration/ServiceCollectionExtensions.cs#L146

Reply:
Done